### PR TITLE
Update Qlogo.cn.xml

### PR DESCRIPTION
**Pull request by renyouguo**

### CODE REVIEW

**Comment by J0WI** [comment]
Please also document `wx.qlogo.cn`

File: src/chrome/content/rules/Qlogo.cn.xml [hunk 1]
```diff
@@ -2,27 +2,32 @@
...
-
+	<target host="t0.qlogo.cn" />
+	<target host="t1.qlogo.cn" />
+	<target host="t2.qlogo.cn" />
+	<target host="t3.qlogo.cn" />
+	<target host="t4.qlogo.cn" />
+	<target host="t5.qlogo.cn" />
+	<target host="t6.qlogo.cn" />
+	<target host="t7.qlogo.cn" />
+	<!--target host="t8.qlogo.cn" /-->
```
Comment (J0WI) [inline src/chrome/content/rules/Qlogo.cn.xml]: please document

Comment (renyouguo) [inline src/chrome/content/rules/Qlogo.cn.xml]: I just tried it and found it ok.

**Comment by J0WI** [comment]
Thanks for clarify.

**Pull request merged**